airflow/gcp/operators/tasks.py

```diff
@@ -616,8 +617,8 @@ def __init__(  # pylint: disable=too-many-arguments
         task: Union[Dict, Task],
         project_id: Optional[str] = None,
         task_name: Optional[str] = None,
-        response_view: Task.View = None,
-        retry: Retry = None,
+        response_view: Optional[enums.Task.View] = None,
```

Comment:
I think the right type is `View` if properly imported.

Reply:
Here is reference on which I've based my imports: https://github.com/googleapis/google-cloud-python/blob/master/tasks/google/cloud/tasks_v2/gapic/enums.py#L87. Due to the fact that it is a class inside another class I suppose it cannot be done in a different way.
It is hard for figure why from `enum` because this information is not well documented: https://github.com/googleapis/google-cloud-python/issues/7503

Reply:
Task.View is an nested class. This is not a common syntax, but I'd rather avoid importing nested classes. This way of importing a class is much more readable what type of object we are dealing with.

Google Cloud Python defines two types of objects:
* types
* enums
For this library, the Task class exists in both types and enums.  For enums, it contains only an internal enum class. For types, this is a real class. It is worth noting that this type is not so obvious, because the Google library has very poor documentation. Of course. We have created to report a separate ticket - https://github.com/googleapis/google-cloud-python/issues/7503

Enum types are also often confused in Google documentation. For example: https://github.com/googleapis/google-cloud-python/pull/8779
In the case of this library, the documentation also very often indicates the wrong location of the enum class.

Reply:
Thanks to help of @mik-laj I've created PR to google-apis with fix to mentioned issue and was successfully merged https://github.com/googleapis/google-cloud-python/pull/9165